### PR TITLE
feat: expose ability to set and reset relayerUrl

### PR DESCRIPTION
**Pull request by esaminu**
Adds `setRelayerUrl` and `resetRelayerUrl` methods to `FastAuthWallet`

### CODE REVIEW

**Review by hcho112 — APPROVED** [review]
LGTM

**Review by MaximusHaximus — APPROVED** [review]
:+1: LGTM :) Thanks @esaminu

File: src/lib/near-fastauth-wallet.ts [hunk 1]
```diff
@@ -188,7 +189,7 @@ const FastAuthWallet: WalletBehaviourFactory<BrowserWallet, { params: FastAuthWa
         const { closeDialog, signedDelegates } = await _state.wallet.requestSignTransactions(arg);
         closeDialog();
         signedDelegates.forEach((signedDelegate) =>
-          fetch(params.relayerUrl, {
```
Comment (Jorslope) [inline src/lib/near-fastauth-wallet.ts:191]: src/lib/near-fastauth-wallet.ts

File: src/lib/near-fastauth-wallet.ts [hunk 2]
```diff
@@ -202,7 +203,7 @@ const FastAuthWallet: WalletBehaviourFactory<BrowserWallet, { params: FastAuthWa
           receiverId: receiverId as string,
         });
 
-        await fetch(params.relayerUrl, {
```
Comment (Jorslope) [inline src/lib/near-fastauth-wallet.ts:205]: _src/lib/near-fastauth-wallet.ts

File: src/lib/near-fastauth-wallet.ts [hunk 3]
```diff
@@ -227,7 +228,7 @@ const FastAuthWallet: WalletBehaviourFactory<BrowserWallet, { params: FastAuthWa
         const { closeDialog, signedDelegates } = await _state.wallet.requestSignTransactions(arg);
         closeDialog();
         signedDelegates.forEach((signedDelegate) =>
-          fetch(params.relayerUrl, {
```
Comment (Jorslope) [inline src/lib/near-fastauth-wallet.ts:230]: src/lib/near-fastauth-wallet.ts

File: src/lib/near-fastauth-wallet.ts [hunk 4]
```diff
@@ -242,7 +243,7 @@ const FastAuthWallet: WalletBehaviourFactory<BrowserWallet, { params: FastAuthWa
             receiverId: receiverId as string,
           });
 
-          await fetch(params.relayerUrl, {
```
Comment (Jorslope) [inline src/lib/near-fastauth-wallet.ts:245]: src/lib/near-fastauth-wallet.ts

File: src/lib/near-fastauth-wallet.ts (hunk 3 above)
Comment (Jorslope) [inline src/lib/near-fastauth-wallet.ts:230]: bdce4d9453947e20cb2a93e96bd280da45f7160600379c7a05232c7d69168752

File: src/lib/near-fastauth-wallet.ts (hunk 4 above)
Comment (Jorslope) [inline src/lib/near-fastauth-wallet.ts:245]: bdce4d9453947e20cb2a93e96bd280da45f7160600379c7a05232c7d69168752

File: src/lib/near-fastauth-wallet.ts (hunk 3 above)
Comment (Jorslope) [reply src/lib/near-fastauth-wallet.ts:230]: bdce4d9453947e20cb2a93e96bd280da45f7160600379c7a05232c7d69168752

**Review by Jorslope — COMMENTED** [review]
src/lib/near-fastauth-wallet.tsbdce4d9453947e20cb2a93e96bd280da45f7160600379c7a05232c7d69168752

**Pull request merged**